src/Illuminate/Support/ServiceProvider.php

```diff
@@ -87,7 +87,7 @@ protected function loadRoutesFrom($path)
      */
     protected function loadViewsFrom($path, $namespace)
     {
-        if (is_array($this->app->config['view']['paths'])) {
+        if (isset($this->app->config['view']) && is_array($this->app->config['view']['paths'])) {
```

Comment:
Safer to do an isset on the full thing, with the paths access?

Reply:
The isset is only on `['view']`, paths is not concerned.

+ See my comment above.

Reply:
IMHO, doing `isset($this->app->config['view'], $this->app->config['view']['paths'])` is overkill as you're not supposed to configure view without paths. However `isset($this->app->config['view'])` is required as view can be not configured (with a fresh install of Lumen for example). And in this case PHP 7.4 throws an error when running `null['paths']`

Reply:
Just do `isset($this->app->config['view']['paths'])`.

Reply:
In which case `config['view']` could be set while `config['view']['paths']` would be null?

Reply:
OK.

Reply:
Done.